MSAL/src/MSALPublicClientApplication.m

```diff
@@ -440,6 +440,11 @@ - (MSALAccount *)accountForUsername:(NSString *)username
                               error:(NSError * __autoreleasing *)error
 {
     MSID_LOG_WITH_CTX_PII(MSIDLogLevelInfo, nil, @"Querying MSAL account for username %@", MSID_PII_LOG_EMAIL(username));
+    if ([NSString msidIsStringNilOrBlank:username])
+    {
+        MSID_LOG_WITH_CTX(MSIDLogLevelWarning, nil, @"username is nil or empty which is unexpected");
```

Comment:
Should we throw an error so the client able to handle it? And the log level should it be error level? Just my opinion.

Reply:
I was little bit struggle here. If putting an error lvl log, I assume we will throw an error. However, we will need to modify the public API which might not be necessary. That's why I only logged a warning instead of error

Reply:
Please ignore above comments, I have updated with the error approach and Utest